modules/cij/analyser.py

```diff
@@ -16,6 +16,34 @@
 import cij.runner
 import cij
 
+_units = {
+    # general
+    '': 1,          # no unit
+    'B': 1,         # bytes
+    'k': 1000,      # kilo
+    'M': 1000**2,   # mega
+    'G': 1000**3,   # giga
+
+    # kibi
```

Comment:
This is awesome. But I think that it should use the IEC prefixes for the 1024^n and the SI for the 1000^n as described here: https://en.wikipedia.org/wiki/Binary_prefix

Reply:
Good point - this is fixed now